### PR TITLE
plural voting implementation in typescript

### PR DESCRIPTION
**Pull request by MartinBenediktBusch**
This PR contains the implementation of the vanilla quadratic voting model and quadratic voting with connection oriented cluster match model in typescript. The implementations can be found in the `qv.ts` and `qv_plural.ts` files. 

Furthermore, the file contains a directory called `__tests__` that contains basic unit test to test the model implementation. The tests are by no means exhausive but the basic functinality of the functions is tested. 

The other files in the repo are standard `ts.` files that are mostly autogenerated.

### CODE REVIEW

File: arb_mvp/src/qv_plural.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    // :param: i: agent_i denotes a participant not equal to a participant called agent_j. 
+    // :param: j: agent_j denotes a participant not equal to a participant called agent_i.
+    // :returns (bool): returns true if two participants share a common group and false otherwise.  
+
+    return groupMemberships[i].some(group => groupMemberships[j].includes(group));
+}
+  
+function K(i: number, group: number[], groupMemberships: number[][], contributions: number[]): number {
+    // Define the weighting function that attenuates the votes of agent i given different group memberships.  
+    // :param: agent_i denotes a participant not equal to a participant called agent_j.
```
Comment (diegoalzate) [inline arb_mvp/src/qv_plural.ts:30]: `agent_j` reference here i think is redundant

**Review by diegoalzate — COMMENTED** [review]
Overall very nice job. left some comments.

File: arb_mvp/src/qv_plural.ts (hunk 1 above)
Comment (MartinBenediktBusch) [reply arb_mvp/src/qv_plural.ts:30]: good point :)

File: arb_mvp/package.json [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+    "@types/jest": "^29.5.10",
+    "@types/mathjs": "^9.4.2",
+    "ts-jest": "^29.1.1"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "author": "",
+  "license": "ISC"
+}
```
[the remark this answers is not in the document]
Comment (MartinBenediktBusch) [reply arb_mvp/package.json]: Thanks for noticing! No clue how all these dependencies made it into the code. I cleaned it up now :)

**Review by diegoalzate — APPROVED** [review]

**Pull request merged**